deploy.py

```diff
@@ -947,7 +947,16 @@ def main(args):
       dest="application",
       action="store_const",
       const="store_latency",
-      help=("Compiles and installs the store_latency example."))
+      help=("Compiles and installs the store_latency example which print "
+            "latency statistics of the persistent store library."))
+  apps_group.add_argument(
+      "--erase_storage",
+      dest="application",
+      action="store_const",
+      const="erase_storage",
+      help=("Compiles and installs the erase_storage example which erases "
+            "the storage. During operation the dongle red light is on. Once "
+            "the operation is completed the dongle green light is on."))
```

Comment:
Does this generalize to the dev kit? I'd guess it's just different LEDs instead of different colors? Might make sense to change the comment, so dev kit users know they are supported.

Reply:
If we plan to keep the `--clear-storage` flag, then dev-kit users should continue using it (as well as dongle users with jlink), it's much simpler to use since it requires only one firmware flash.

The `erase_storage` binary is only for dongle users that only have DFU to flash firmware. This binary is a work-around for them not being able to use `--clear-storage`.

Reply:
Okay, gotcha!